app/code/Magento/Review/Api/GetProductRatingSummaryInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Review\Api;
+
+/**
+ * Interface to get product rating summary
+ */
+interface GetProductRatingSummaryInterface
+{
+    /**
+     * Returns the product rating summary
+     *
+     * @param int $productId
+     * @param int $storeId
+     *
+     * @return int Rating summary
+     */
+    public function execute(int $productId, int $storeId = 0): int;
```

Comment:
Why is average rating represented as int?

Reply:
Was checking the following implementation:
https://github.com/magento/magento2/blob/e42ed8dcf89ce65d7ba4af489fc6e6cd764e161b/app/code/Magento/Review/Model/ResourceModel/Review/Summary.php#L60-L64

But, never mind as I got rid of this interface and the custom implementation.